### ISSUE TITLE
if glsl has some kind of bug, but it don't throw error!

### ISSUE THREAD

**Issue by yyc-git** [maierfelix/webgpu#10]
Hello! 
I find that if there are some kind of the error in the glsl,  just exit and throw no error after run the program! so it's difficult to find the error! I have to comment the code in glsl one by one to find the error!

some error like this will throw no error:
ray-closest-hit.rchit:
```glsl
...
//this line cause bug, because two SceneDesc!
layout(scalar, set = 1, binding = 0) buffer SceneDesc { InstanceData i[]; } SceneDesc;

should fix to:
layout(scalar, set = 1, binding = 0) buffer SceneDesc { InstanceData i[]; } sceneDesc;
```

Could you explain why and How to make it throw error???Thanks very much

**Comment by maierfelix** [comment]
I didn't put too much effort in the error throw system to propagate errors from dawn over to JavaScript. This is indeed a problem and should be fixed.

Edit: To spot errors early, I use GLSL Validator in Sublime Text 3, you might want to give it a shot too. I use [this](https://gist.github.com/maierfelix/98371a3d774087177f178af378855d77) custom script to support latest Ray-Tracing shaders.

**Comment by yyc-git** [comment]
Thanks very much!
The error throw system is important for debug glsl!!!

**Comment by maierfelix** [comment]
See my updated answer above - or you can also give [glslangValidator](https://github.com/KhronosGroup/glslang/releases) a try.

**Comment by yyc-git** [comment]
Thanks very much! but I'm using vs code.

**Comment by yyc-git** [comment]
@maierfelix Thanks for your help, I can use glsl lint in vscode now:
1、install glslangValidator by download [glslang-master-windows-x64-Release.zip](https://github.com/KhronosGroup/glslang/releases)(My OS is win7)
2、unzip
3、add its bin dir to environment path
e.g.
C:\Users\Administrator\Downloads\glslang-master-windows-x64-Release\bin
4、in vs code, install “Shaderc GLSL Linter” extension

then you can has lint tip in glsl(include .rgen, .frag, .vert, ...)!!!

Thank you!

**Comment by maierfelix** [comment]
@yyc-git this should be fixed now in `0.1.7`

**Comment by yyc-git** [comment]
@maierfelix Thanks very much!
